infrastructure/index.ts

```diff
@@ -336,3 +336,4 @@ async function createAliasRecord(
 export const originBucketWebsiteDomain = originBucket.websiteDomain;
 export const originBucketWebsiteEndpoint = originBucket.websiteEndpoint;
 export const cloudFrontDomain = cdn.domainName;
+export const cloudFrontDistributionUrn = cdn.urn;
```

Comment:
I added this stack output because we are currently hard-coding the distribution URN [here](https://github.com/pulumi/docs/blob/master/scripts/run-pulumi.sh#L26). Once the stack output is available, we could replace the hard-coded value with this.

Reply:
The CDN is protected, so should never change, but even if it did, this approach should still hold. Makes sense to me. 👍